servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -69,7 +69,7 @@ quarkus.oidc.enabled=false
 ## Quarkus swagger settings
 # fixed at buildtime
 quarkus.swagger-ui.always-include=true
-quarkus.swagger-ui.enable=false
+quarkus.swagger-ui.enable=true
```

Comment:
Does this mean it will be on by default? I'd like to include but have off by default.

Reply:
I'm a bit confused by the configuration keys at https://quarkus.io/guides/openapi-swaggerui#quarkus-smallrye-openapi_configuration

Should we enable, but only include for dev mode?

Reply:
`quarkus.swagger-ui.always-include` adds teh swagger UI to the package
`quarkus.swagger-ui.enable` actually adds a route to the running application.

However both are 'build-time' configs so they are respected only at build time and can't be turned on in the graalvm image. 

This change enables the UI at build time so it will be available in the docker image. Lately I have been using the docker image for testing UI/CLI/Spark so I find the swagger ui activated to be rather useful but as it becomes more mature I think we would want to take it out of the docker image.

So we can:
1) take this change and leave swagger ui always on
2) change this so that the swagger ui is only included and available when running `quarkus:dev`

I prefer to leave the swagger ui in the docker image for at least a few months but am not that bothered if someone else feels diffferently.